ingest/lambdas/bitstreams-downloader.ts

```diff
@@ -0,0 +1,78 @@
+import { z } from 'zod';
+
+import { s3 } from '../../lib/aws-clients';
+import * as dspaceClient from '../../lib/dspace-client';
+import { S3ObjectLocation, safeGetObjectJson } from '../../lib/s3-utils';
+import lambdaClient from '../../lib/lambda-client';
```

Comment:
This produces a ts error but other than converting that file to typescript could I just put a ts-ignore here?

Reply:
`// @ts-expect-error`, so that we remember to remove that comment once it's moved to TS.